es6.js

```diff
@@ -7,35 +7,33 @@ module.exports = {
   plugins: ['@babel', 'html'],
   extends: ['airbnb-base', 'plugin:import/typescript'],
   env: {
-    es6: true,
+    es2021: true,
```

Comment:
Note from Jordi:
Use latest env

Reply:
```es6 - enable all ECMAScript 6 features except for modules (this automatically sets the ecmaVersion parser option to 6).

es2021 - adds all ECMAScript 2021 globals and automatically sets the ecmaVersion parser option to 12.```

The only difference I've found with this change is that it won't barf about using BigInt.

Also, es2021: true automatically set the ecmaVersion parser option to 12, but @FSVetaz  explicitly changed to ecmaVersion to `latest`. Would it be better to keep it consistent/clear what is happening there?

Reply:
When I do npx eslint --print-config file.js > eslint.txt, this is the difference if I leave out ecmaVersion altogether ( i get ecmaVersion 2018 (aka 9): 
![image](https://user-images.githubusercontent.com/77301861/215884574-9dde682b-9c17-48c6-abad-967960a89923.png)

I thought latest would be alright, but we could change it to 12 to have it be consistent with the env.

Reply:
I'll change it to 12.

Reply:
I say we just remove it since setting es2021 sets it automatically

Reply:
Something is overriding it that is setting it to 2018. I'm not sure what though

Reply:
https://github.com/airbnb/javascript/blob/1eadb93e377da1e56c3f91f26610e5d0a00738a9/packages/eslint-config-airbnb-base/index.js airbnb is overriding it.

Reply:
Okay, we can explicitly set ecmaVersion to `12` then.